enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -200,7 +200,7 @@ const (
 	guestVFSMountDir = "/vfs"
 
 	// How long to allow for the VM to be finalized (paused, outputs copied, etc.)
-	finalizationTimeout = 10 * time.Second
+	finalizationTimeout = 3 * time.Minute
```

Comment:
This is used for `remove()` and for syncing the workspace drive so I think we should keep the previous value